src/Mnemonic.js

```diff
@@ -91,10 +91,45 @@ export default class Mnemonic {
         }
 
         return CACHE.privateKeyConstructor(
+            // eslint-disable-next-line deprecation/deprecation
             await this._mnemonic.toPrivateKey(passphrase)
         );
     }
 
+    /**
+     * Recover an Ed25519 private key from this mnemonic phrase, with an
+     * optional passphrase.
+     *
+     * @param {string} [passphrase]
+     * @returns {Promise<PrivateKey>}
+     */
+    async toEd25519PrivateKey(passphrase = "") {
```

Comment:
Q: should the path be exposed in these methods.
Not too familiar with the structure. This is most devs will call correct?
Then if they want to customize do they have to depend on the cryptography package specifically?

Reply:
The path should be provided here, that's a mistake on my part. I already added the necessary support to `packages/cryptography/src/Mnemonic`, but forgot to update this file as well.